README.md

```diff
@@ -26,7 +26,8 @@ Additionally:
 
 Collaborations with:
 
-* [Institut für Systemsoftware at Johannes Kepler University Linz](http://ssw.jku.at)
+* [Institut für Systemsoftware at Johannes Kepler University
+   Linz](http://ssw.jku.at)
```

Comment:
This change for instance degrades readability, because 80 chars is arbitrarily short.